src/50_Visual_Effects/Basics_of_orientation_effects.html

```diff
@@ -0,0 +1,304 @@
+      ]
+    }
+    </script>
+  </amp-animation>
+
+  <!--
+  ## Tilt based translation animations
+
+  Let's create a tilt based animation where the clock's hand moves 
+  as you tilt the device along a specific axis.
+  -->
+
+  <!--
+  The first animation is along the `beta` axis, to see this in action rotate your phone along the y axis. 
+
+  Here you can see that we are using the `beta` event to progress through the animation.
+  -->
+
+  <div id="clock-scene-1" class="clock-scene">
+    <amp-orientation-observer
+      beta-range="0 180"
+      on="beta:clockAnim1.seekTo(percent=event.percent)"
+      layout="nodisplay">
+    </amp-orientation-observer>
+    <amp-img layout="responsive" width=2 height=1.5 src="/img/clock.jpg">
+      <div class="clock-hand rotate-1"></div>
+    </amp-img>
+  </div>
+
+  <!--
+  The animation in this case is a rotation of the clock hand from 
+  -180 degrees to 180 degrees to allow the hand to start and end at 
+  the 12 o' clock position.
+  -->
+
+  <amp-animation id="clockAnim1" layout="nodisplay">
+    <script type="application/json">
+      {
+      "duration": "3s",
+      "fill": "both",
+      "direction": "alternate",
+      "animations": [
+        {
+          "selector": "#clock-scene-1 .rotate-1",
+          "keyframes": [
+            { "transform": "rotate(-180deg)" },
+            { "transform": "rotate(180deg)" }
+          ]
+        }
+      ]
+    }
+    </script>
+  </amp-animation>
+
+  <!--
+  The second animation is along the `gamma` axis, to see this in action rotate your phone along the x axis:
+  
+  Here you can see that we are using the `gamma` event to progress through the animation.
+  -->  
+
+  <div id="clock-scene-1" class="clock-scene">
+    <amp-orientation-observer
+      on="gamma:clockAnim2.seekTo(percent=event.percent)"
+      layout="nodisplay">
+    </amp-orientation-observer>
+    <amp-img layout="responsive" width=2 height=1.5 src="/img/clock.jpg">
+      <div class="clock-hand rotate-2"></div>
+    </amp-img>
+  </div>
+
+  <!--
+  The animation in this case is a rotation of the clock hand from 
+  -270 degrees to 90 degrees to allow the hand to start and end at 
+  the 12 o' clock position. The motion here is limited from the 
+  9 o' clock position to the 3 o' clock position.
+  -->
+
+  <amp-animation id="clockAnim2" layout="nodisplay">
+    <script type="application/json">
+      {
+      "duration": "3s",
+      "fill": "both",
+      "direction": "alternate",
+      "animations": [
+        {
+          "selector": "#clock-scene-1 .rotate-2",
+          "keyframes": [
+            { "transform": "rotate(-270deg)" },
+            { "transform": "rotate(-90deg)" }
+          ]
+        }
+      ]
+    }
+    </script>
+  </amp-animation>
+
+  <!--
+  The third animation is along the `alpha` axis, to see this in action rotate your phone along the horizontal axis:
+
+  Here you can see that we are using the `alpha` event to progress through the animation.
```

Comment:
Is this sentence still needed after the previous one?

Reply:
Removed.